python/extensions/python.bzl

```diff
@@ -17,35 +17,117 @@
 load("@rules_python//python:repositories.bzl", "python_register_toolchains")
 load("@rules_python//python/extensions/private:interpreter_hub.bzl", "hub_repo")
```

Comment:
minor: having `_interpreter_hub_repo = "hub_repo"` here would make the code below more readable.

Reply:
I think "interpreter hub" and "toolchain hub" are sufficiently clear.

"interpreter hub" isn't great, but we couldn't think of a better name for it. The interpreter hub name is internal, so we can rename it if we think of a better name.